lib/Jsrt/JsrtHelper.cpp

```diff
@@ -89,7 +89,7 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
         pthread_key_create(&s_threadLocalDummy, DISPOSE_CHAKRA_CORE_THREAD);
 #endif
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) || !defined(_WIN32)
```

Comment:
> Reason I'm still working with shared lib is because static lib is unstable (hangs)

You are referring this for using from outside right? I will dedicate some personal time if you share me a repo/branch at that particular state. Currently static part is the only part is working cross platform (osx etc.). Whatever the problem is there, I'm not sure it is because of the CC side. Actually by enabling this part; you are using the static lib interface for shared lib. So, it is interesting that it doesn't hang.

Please feel free to enable this trick until I introduce lazy init to PAL though. I really appreciate if you can add a todo note to me there.